cpp/src/fil/fil.cu

```diff
@@ -612,7 +627,8 @@ void tl2fil_common(forest_params_t* params, const tl::Model& model,
   }
   if (std::string(param.pred_transform) == "sigmoid") {
     params->output = output_t(params->output | output_t::SIGMOID);
-  } else if (std::string(param.pred_transform) != "identity") {
+  } else if (std::string(param.pred_transform) != "identity" &&
```

Comment:
These checks are performed when checking for the supported transformations, and are not needed here.